lesson-6-object/index.js

```diff
@@ -3,23 +3,75 @@ void readmeLoader();
 
 //TASK-1
 
+let person = {
+    firstName: 'Ali',
+    lastName: 'Kenzo',
+    age: '23',
+    hello: function() {
+        console.log('Hello my name is: ' + this.firstName + ' ' + this.lastName);
+    }
+}
+console.log('TASK-1', person.hello());
+
 //TASK-2
 
+person.familyMembers = ['Veli', 'Hasan'];
+console.log('TASK-2', person);
+
 //TASK-3
 
+person.house = {
+    country: 'Nepal',
+    city: 'Kathmandu',
+    zipCode: '36950'
+}
+console.log('TASK-3', person);
+
 //TASK-4
 
+function addField(name, value) {
+    person[name] = value;
+}
+addField('weight', '60kg');
+console.log('TASK-4', person);
```

Comment:
**_`addField`_** should be a field of **`person`** object similar like **_`hello`_** field. Please change